binnacle/lib/providers/navigation_provider.dart

```diff
@@ -13,30 +14,53 @@ class NavigationProvider {
   Vector2 _start;
   Vector2 _end;
   RouteModel _route;
-  BehaviorSubject eventBus = BehaviorSubject();
+  BehaviorSubject<NavigationEvent> eventBus = BehaviorSubject();
+  ReplaySubject<PositionModel> positionHistory;
+
+  List<LatLng> course;
```

Comment:
Is there a reason this is of type LatLng instead of Position?

Reply:
Yes. Google maps plugin has no concept of our position model, nor should it. It doesn't care about speed. So any input should be handled with its own models, which in this case is LatLng.

Additionally, I focused on the map screen being at the scope of Google maps, so most conversions should happen outside of the widget to provide it just LatLng inputs.

Reply:
Understood.